include/nbl/asset/filters/CSwizzleAndConvertImageFilter.h

```diff
@@ -69,13 +69,13 @@ class CSwizzleAndConvertImageFilterBase : public CSwizzleableAndDitherableFilter
 						{						
 							constexpr auto maxChannels = 4;
 							decodeBufferType decodeBuffer[maxChannels] = {};
-							encodeBufferType encodeBuffer[maxChannels] = {};
 
 							if constexpr (kInFormat!=EF_UNKNOWN)
-								base_t::template onDecode<kInFormat,encodeBufferType>(state, srcPix, decodeBuffer, encodeBuffer, blockX, blockY);
+								base_t::template onDecode<kInFormat>(state, srcPix, decodeBuffer, blockX, blockY);
 							else
-								base_t::template onDecode<encodeBufferType>(rInFormat, state, srcPix, decodeBuffer, encodeBuffer, blockX, blockY);
-							state->normalization.prepass(encodeBuffer,readBlockPos*blockDims+commonExecuteData.offsetDifferenceInTexels,blockX,blockY,4u/*TODO: figure this out*/);
+								base_t::template onDecode<encodeBufferType>(rInFormat, state, srcPix, decodeBuffer, blockX, blockY);
+
+							state->normalization.prepass(decodeBuffer,readBlockPos*blockDims+commonExecuteData.offsetDifferenceInTexels,blockX,blockY,4u/*TODO: figure this out*/);
```

Comment:
all changes to this file from this commit break it: https://github.com/Devsh-Graphics-Programming/Nabla/pull/435#discussion_r1278978388

Reply:
ok you've moved the temp